src/App.js

```diff
@@ -1,24 +1,99 @@
-import logo from './logo.svg';
 import './App.css';
+import Header from './components/Header';
+import {useState} from 'react';
+import axios from 'axios';
+import Weather from './components/Weather';
+import WeatherDescription from './components/WeatherDescription';
 
 function App() {
+
+    const [weatherData, setWeatherData] = useState({})
+    const [location, setLocation] = useState('')
+    const [backgroundImg, setBackgroundImg] = useState("startingBackground")
+
+//API CALL
+
+    const CallWeather = (event) => {
```

Comment:
may want to consider putting the user's input into a separate component and perhaps use a form submit event listener instead of a click event.

Reply:
Thats something i'd like to do for sure. While what I have is functional having it in a separate component would be helpful especially if I were to be reusing it in the future.